Makefile

```diff
@@ -211,6 +213,7 @@ dev-dracon: deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
 		  --values ./deploy/dracon/values/dev.yaml \
 		  --create-namespace \
 		  --namespace $(DRACON_NS) \
+          --version $$(echo "${DRACON_VERSION}" | sed 's/^v//')
```

Comment:
the line has a different format than the rest of the command and it's missing a slash at the end. there is a small bug at the next line that's also missing a slash at the end, unrelated to this PR but I think we should fix it since we are working here.